docs/01-overview/main-areas/observability/obsv-04-telemetry-in-kyma.md

```diff
@@ -0,0 +1,392 @@
+
+Fluent Bit comes with a pipeline concept, which supports flexible combination of inputs with outputs and filtering in between - for details, see [Fluent Bit: Output](https://docs.fluentbit.io/manual/concepts/data-pipeline/output).
+Kyma's telemetry component brings a predefined setup of the Fluent Bit DaemonSet and a base configuration, which assures that the application logs of the workloads in the cluster are processed reliably and efficiently. Additionally, the telemetry component provides a Kubernetes API called `LogPipeline` to configure outputs with some filtering capabilities.
+
+![Pipeline Concept](./assets/telemetry-logging-pipelines.drawio.svg)
+
+1. A central `tail` input plugin reads the application logs.
+
+2. The application logs are enriched by a `kubernetes` filter. Then, for every `LogPipeline` definition, a `rewrite_tag` filter is generated, which uses a dedicated `tag` with name `<logpipeline>.*`, followed by the custom configuration defined in the `LogPipeline` resource. You can add your own filters to the default filters.
+
+3. Based on the default and custom filters, you get the desired output for each `LogPipeline`.
+
+This approach assures a reliable buffer management and isolation of pipelines, while keeping flexibility on customizations.
+
+### Telemetry Operator
+
+The LogPipeline resource is managed by the `Telemetry Operator`, a typical Kubernetes operator responsible for managing the custom parts of the Fluent Bit configuration.
+
+![Operator resources](./assets/telemetry-operator-resources.drawio.svg)
+
+The `Telemetry Operator` watches all Log Pipeline resources and related secrets. Whenever the configuration changes, it validates the configuration (with a [validating webhook](https://kubernetes.io/docs/reference/access-authn-authz/extensible-admission-controllers/)) and generates a new configuration for the Fluent Bit DaemonSet, where several ConfigMaps for the different aspects of the configuration are generated. Furthermore, referenced Secrets are copied into one Secret that is mounted to the DaemonSet as well.
+
+## Set up the LogPipeline
+
+### Step 1: Create a LogPipeline
+
+1. To ship application logs to a new output, create a resource file of kind LogPipeline:
+
+   ```yaml
+   kind: LogPipeline
+   apiVersion: telemetry.kyma-project.io/v1alpha1
+   metadata:
+     name: http-backend
+   spec:
+     output:
+       custom: |
+         Name               http
+         Host               https://myhost/logs
+         Tls                On
+   
+   ```
+
+   An output is a data destination that can be configured by a [Fluent Bit output](https://docs.fluentbit.io/manual/pipeline/outputs) of the relevant type. Here, the used output type is `http`, which is configured with the relevant parameters.
+
+2. To create the instance, apply the resource file in your cluster.
+
+   ```bash
+   kubectl apply -f path/to/my-log-pipeline.yaml
+   ```
+
+3. Check that the status of the pipeline in your cluster is `Ready`:
```

Comment:
When do we say "pipeline" generically, and when "LogPipeline" as Kubernetes resource? 
Or is it all the same? In that case, we should consistently just one term.

Reply:
```suggestion
3. Check that the status of the LogPipeline in your cluster is `Ready`:
```